scripts/test_config.yaml

```diff
@@ -1,54 +1,59 @@
 # config.yaml
+potentials:
+  - provider: "physics-ml"
+    model_name: "MACE"
+    precision: "64"
+    position_scaling: 10.0
+    output_scaling: 4.184 * 627
+    model_path: "/guardowl/tests/data/physics-ml/mace"
+    device: "cuda"
```

Comment:
I added the `device` argument as the default used in `physicsml` is cpu, so we need to specify this when running on GPU otherwise the kernel will die due to an incorrect version of pytorch.

I am tempted to automate this and set the device as `cuda` if a GPU is detected as this will be the primary mode of execution.

Reply:
great, thanks! 

I prefer to pass this explicitly, though I think it should have a seperate entry in the `yaml that is used for all defined potentials.

Something like

```yaml
device:
    device_name: 'cuda'
    properties:
       precision : 'mixed'
```